systemtest/src/test/resources/connect-build/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG KAFKA_VERSION
+ARG BASE_IMAGE=quay.io/strimzi/kafka:latest-kafka-${KAFKA_VERSION}
```

Comment:
How does this work for release tests? It should use the GA / RC image in such case.

Reply:
In the AZP I'm passing it via `BASE_IMAGE: 'localhost:5000/$(docker_org)/kafka:$(docker_tag)-kafka-KAFKA_VERSION'` env, which should point to images that were build in one of the previous steps.

Reply:
But when running the tests for releases, it does not build any images.

Reply:
Yeah, I will fix that. Good catch. Thanks

Reply:
So I changed it to `BASE_IMAGE: '$(docker_registry)/$(docker_org)/kafka:$(docker_tag)-kafka-KAFKA_VERSION'`. That should fix the issue with release pipelines. WDYT?

Reply:
I'm afraid I cannot say it just like that. You would need to try it.